packages/ag-charts-enterprise/src/features/annotations/annotations.ts

```diff
@@ -650,350 +668,6 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
-        );
-        this.updateToolbarColorPickerFill(
-            AnnotationOptions.TextColor,
-            datum?.getDefaultColor(AnnotationOptions.TextColor),
-            datum?.getDefaultOpacity(AnnotationOptions.TextColor)
-        );
-    }
-
-    private updateToolbarFontSize(fontSize: number | undefined) {
-        this.ctx.toolbarManager.updateButton('annotationOptions', AnnotationOptions.TextSize, {
-            label: fontSize != null ? String(fontSize) : undefined,
-        });
-    }
-
-    private updateToolbarLineStyleType(item: MenuItem<AgAnnotationLineStyleType>) {
-        this.ctx.toolbarManager.updateButton('annotationOptions', AnnotationOptions.LineStyleType, {
-            icon: item.icon,
-        });
-    }
-
-    private updateToolbarStrokeWidth(item: MenuItem<number>) {
-        this.ctx.toolbarManager.updateButton('annotationOptions', AnnotationOptions.LineStrokeWidth, {
-            label: item.label,
-            strokeWidth: item.value,
-        });
-    }
-
-    private onTextSizeMenuPress(item: MenuItem<number>, datum?: AnnotationProperties) {
-        if (!hasFontSize(datum)) return;
-
-        const fontSize = item.value;
-        this.setFontSizeAndDefault(datum.type, fontSize);
-        this.textSizeMenu.hide();
-        this.updateToolbarFontSize(fontSize);
-    }
-
-    private onLineStyleTypeMenuPress(item: MenuItem<AgAnnotationLineStyleType>, datum?: AnnotationProperties) {
-        if (!hasLineStyle(datum)) return;
-
-        const type = item.value;
-        this.setLineStyleTypeAndDefault(datum.type, type);
-        this.lineStyleTypeMenu.hide();
-        this.updateToolbarLineStyleType(item);
-    }
-
-    private onLineStrokeWidthMenuPress(item: MenuItem<number>, datum?: AnnotationProperties) {
-        if (!hasLineStyle(datum)) {
-            return;
-        }
-
-        const strokeWidth = item.value;
-        this.setLineStyleWidthAndDefault(datum.type, strokeWidth);
-        this.lineStrokeWidthMenu.hide();
-        this.updateToolbarStrokeWidth(item);
-    }
-
-    private onAnnotationsMenuPress(
-        event: _ModuleSupport.ToolbarButtonPressedEvent<AgToolbarAnnotationsButtonValue>,
-        item: MenuItem<AnnotationType>
-    ) {
-        const { toolbarManager } = this.ctx;
-
-        toolbarManager.toggleButton('annotations', event.id, {
-            active: true,
-        });
-        toolbarManager.updateButton('annotations', event.id, {
-            icon: item.icon,
-        });
-
-        this.beginAnnotationPlacement(item.value);
-        this.annotationMenu.hide();
-
-        this.removeAmbientKeyboardListener?.();
-        this.removeAmbientKeyboardListener = this.ctx.interactionManager.addListener(
-            'keydown',
-            (e) => this.handleAmbientKeyboardEvent(e),
-            InteractionState.All
-        );
-    }
-
-    private handleAmbientKeyboardEvent(e: _ModuleSupport.KeyInteractionEvent<'keydown'>) {
```

Comment:
This listener is no longer needed. Previously it handled cancelling creation when the user has picked from the menu but not yet clicked on the chart, i.e. the focus was not on the chart.

That is now fixed elsewhere in interactions so the default `onKeyDown()` handles this.